CHANGELOG.md

```diff
@@ -3,14 +3,248 @@
+ - The visualizer now shows coordinate arrows for all rigid transforms within the view. [#2577](https://github.com/rerun-io/rerun/pull/2577)
+ - Linestrips and oriented bounding boxes can now be logged via batch APIs in python.
+   - See: `log_linestrips_2d`, `log_linestrips_3d`, [#2822](https://github.com/rerun-io/rerun/pull/2822) and `log_obbs` [#2823](https://github.com/rerun-io/rerun/pull/2823)
+ - New optional `flush_timeout` specifies how long Rerun will wait if a TCP stream is disconnected during a flush. [#2821](https://github.com/rerun-io/rerun/pull/2821)
+    - In Rust, `RecordingStream::connect` now requires `flush_timeout` specified as an `Option<Duration>`.
+      - To keep default behavior, this can be specified using the `rerun::default_flush_time()` helper.
+    - In Python `flush_init_sec` is now an optional argument to `rr.connect()`
+
+### Ongoing Refactors
+ - There have been a number of significant internal changes going on during this release with little visible impact.
+   This work will land across future releases, but is highlighted here since much of it is visible through the
+   changelog.
+   - The layout of the Viewer is now controlled by a Blueprint datastore. In the future this will allow for direct API
+    control of the layout and configuration of the Viewer. A very early prototype of this functionality is available
+    via the `rerun.experimental` module in Python.
+   - An entirely new code-generation framework has been brought online for Rust and Python. This will eventually enable
+    new object-centric APIs with a more scalable, consistent, and ergonomic experience.
+   - Bringup of C++ support is now underway and will eventually become our third officially supported SDK language.
+
+### Known Regressions
+- Due to the Blueprint storage migration, blueprint persistent on web is currently broken. Will be resolved in: [#2579][https://github.com/rerun-io/rerun/issues/2579]
+
+### In Detail
+#### 🐍 Python SDK
+- Clean up warnings printed when `rr.init` hasn't been called [#2209](https://github.com/rerun-io/rerun/pull/2209)
+- Normalise Python typing syntax to 3.8+ [#2361](https://github.com/rerun-io/rerun/pull/2361)
+- Simpler, sturdier stateful time tracking in both SDKs [#2506](https://github.com/rerun-io/rerun/pull/2506)
+- Fix not taking np.array for single colors [#2569](https://github.com/rerun-io/rerun/pull/2569)
+- Expose batch APIs for linestrips [#2822](https://github.com/rerun-io/rerun/pull/2822)
+- Expose batch APIs for oriented bounding boxes [#2823](https://github.com/rerun-io/rerun/pull/2823)
+- Improve `log_pinhole` and support non-RDF pinholes [#2614](https://github.com/rerun-io/rerun/pull/2614)
+
+#### 🦀 Rust SDK
+- Add example for adding custom Space Views [#2328](https://github.com/rerun-io/rerun/pull/2328)
+- Simpler, sturdier stateful time tracking in both SDKs [#2506](https://github.com/rerun-io/rerun/pull/2506)
```

Comment:
This deserves a spot in the overview imo; a few users have directly asked for this to be fixed in the past.